static/scss/partials/main.scss

```diff
@@ -34,7 +34,8 @@
   --successGreen: #3FE1B0; 
   --white: #fff;
   --background: #F5F7F9;
-  --dashboardMaxWith: calc(830px + 3rem);
+  // --dashboardMaxWith: calc(830px + 3rem);
```

Comment:
The dashboard width was revised in the last few iterations. This should give everything much more room to fit.

Reply:
We could use `$content-lg (928px)` instead? 830px + 3rem is ~878px, could be close enough.

Reply:
Great call!